4.js

```diff
@@ -1,10 +1,7 @@
-import PI from "./Math";
-import sqrt from "./Math";
-import square from "./Math";
-
 var arg1 = process.argv[2];
 var arg2 = process.argv[3];
```

Comment:
基本letかconstにして．
varよりもletのほうがスコープが小さいので．
```suggestion
const arg1 = process.argv[2];
```